packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -1017,4 +1058,79 @@ describe('Right Click Menu integration', () => {
       });
     });
   });
+
+  describe('right click menu: multiple elements selected', () => {
+    it('should duplicate all selected elements', async () => {
+      await addText({
+        y: 300,
+        fontSize: 40,
+        content: '<span style="color: #10ff01">Another Text Element</span>',
+      });
+      const image = await addEarthImage();
+      const shape = await addShape({
+        backgroundColor: {
+          color: {
+            r: 203,
+            g: 103,
+            b: 103,
+          },
+        },
+      });
+
+      const imageFrame = fixture.editor.canvas.framesLayer.frame(image.id).node;
+      const shapeFrame = fixture.editor.canvas.framesLayer.frame(shape.id).node;
+
+      // select multiple targets
+      await clickOnTarget(imageFrame);
+      await clickOnTarget(shapeFrame, 'Shift');
+
+      // multiple elements should be selected
+      const { initialElements, selectedElements } = await fixture.renderHook(
+        () =>
+          useStory(({ state }) => ({
+            selectedElements: state.selectedElements,
+            initialElements: state.currentPage.elements,
+          }))
+      );
+
+      expect(selectedElements.length).toBe(2);
+      expect(initialElements.length).toBe(4);
+
+      // open right click menu
+      await rightClickOnTarget(imageFrame);
```

Comment:
here I'm right clicking on the image but you can technically right click anywhere in the selection area

Decided against testing that since selection using the multi select area is tested in the multi select karma tests. thoughts?

Reply:
yea I'm fine with that. I think it's kinda one half dozen the other. Seems like the fixture makes it easier to grab this node vs the multi select area so I'd stick with this since it's more convenient